core/hibernate-search/src/main/java/io/thorntail/hibernate_search/HibernateSearchFraction.java

```diff
@@ -0,0 +1,6 @@
+package io.thorntail.hibernate_search;
+
+/**
+ * @author kg6zvp
+ */
+public class HibernateSearchFraction { }
```

Comment:
Are fraction classes needed in v4?

Reply:
Yes, this isn't needed

Reply:
This is to force jar packaging

Reply:
package-info.java isn't enough?

Reply:
Will verify when I get access to my computer. I believe I had tried that initially, though.